app/services/attachments/destroy_service.rb

```diff
@@ -28,7 +28,19 @@ def execute # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
         destroyed_attachments.append(associated_attachment)
       end
 
-      @attachment.destroy
+      attachment_destroyed = @attachment.destroy
+
+      if attachment_destroyed
+        @attachable.project.namespace.create_activity key: 'namespaces_project_namespace.samples.attachment.destroy',
+                                                      owner: current_user,
+                                                      trackable_id: @attachable.id,
+                                                      parameters: {
+                                                        sample_name: @attachable.puid,
```

Comment:
**nitpick (blocking):** Can you change this parameter to `sample_puid`

Reply:
Updated in 032cb1c08ee993dee5d18b74ef198b03bafe07b3